Dockerfile.pipelines

```diff
@@ -18,9 +21,21 @@ RUN apt-get -y dist-upgrade
 RUN apt-get -y install wget bzip2 git make
 
 # Default gcc version to install; can be overridden in Jenkinsfile
```

Comment:
probably you mean something else instead of Jenkinsfile?

Reply:
Oops, yes :-)